src/scripts/circleci-job-canceller/long_job_canceller.py

```diff
@@ -30,6 +32,23 @@ def get_workflow_started_by(current_workflow, headers):
     return username
 
 
+def get_workflow_pending_approval_jobs(workflow_id, headers):
```

Comment:
If we passed in the `ignore` parameter here, we could even paginate through the responses, and upon finding _any_ job which is pending but not named incorrectly, we could terminate 🤷